src/Npgsql/NpgsqlBinaryImporter.cs

```diff
@@ -15,7 +17,7 @@ namespace Npgsql
     /// <remarks>
     /// See http://www.postgresql.org/docs/current/static/sql-copy.html.
     /// </remarks>
-    public sealed class NpgsqlBinaryImporter : ICancelable
+    public sealed class NpgsqlBinaryImporter : ICancelable, IAsyncDisposable
```

Comment:
Also, it's odd for this class to directly implement IAsyncDisposable but not IDisposable. Move the implementation of IDisposable to here (ICancelable should stay but not implement anything).

Same with NpgsqlBinaryExporter (as @YohDeadfall wrote above) - make it directly implement both disposable interfaces.